source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -1,7 +1,4 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-.. meta::
-  :description: The ossec.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here. 
-
 
```

Comment:
Missing meta description. Please, add this element and make sure it corresponds with the content of the section.

```suggestion


```